com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.RenderGraph.cs

```diff
@@ -294,9 +294,9 @@ TextureHandle DynamicExposurePass(RenderGraph renderGraph, HDCamera hdCamera, Te
                     }
                     else
                     {
-                        passData.tmpTarget1024 = builder.CreateTransientTexture(new TextureDesc(1024, 1024, true, false)
+                        passData.tmpTarget1024 = builder.CreateTransientTexture(new TextureDesc(1024, 1024, false, false)
```

Comment:
I am _sure_ this was not set to do hw dyn res to start with, so it makes me think lots of stuff got messed up during the render graph transition; let me know if you want an extra pair of eyes but I think we need to do a pass on all the new TextureDesc;

Reply:
So there are two ways of fixing it for hardware:
1) We disable DRS on this target, and carry on. This would make exposure match %100 software.
2) We keep this as a DRS target, but we have to change the code internally, and also the dispatch size must match the size of this target (its hard coded to 1024, which causes just a piece of the frame to be put in the exposure input).

Let me kow your thoughts. I like option # 1 better, because exposure remains consistenish between software and hardware DRS

Reply:
It should be 1 and it was 1 before rendergraph I am sure :-) 

Hence why I am saying it might be that we might have made similar port issues from before rendergraph to rendergraph (and we should make a pass to check if it is indeed the case)

Reply:
💪

Reply:
Damn, the carnage continues!
It's not my fault, it's the reviewers :p